test-app-reactive/src/test/kotlin/uk/gov/justice/digital/hmpps/testappreactive/integration/auth/customizer/AuthorizeExchangeCustomizerTest.kt

```diff
@@ -2,13 +2,18 @@ package uk.gov.justice.digital.hmpps.testappreactive.integration.auth.customizer
 
 import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Test
+import org.springframework.boot.test.context.SpringBootTest
 import org.springframework.boot.test.context.TestConfiguration
 import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Import
 import uk.gov.justice.digital.hmpps.testappreactive.integration.IntegrationTestBase
 import uk.gov.justice.hmpps.kotlin.auth.dsl.ResourceServerConfigurationCustomizer
 
 @Import(AuthorizeExchangeCustomizerTest.CustomizerConfiguration::class)
+@SpringBootTest(
```

Comment:
since I added a `ResourceServerConfiguration` to the test app this needs to be overridden in the test

Reply:
Good coment - but maybe add as a comment in the source code too? Otherwise it's not obvious why that property is being set.